src/draft-ietf-rpp-requirements.md

```diff
@@ -266,6 +266,18 @@ A> TODO: [Issue #15](https://github.com/ietf-wg-rpp/rpp-requirements/issues/15)
 
 **R9.10** RPP MUST allow users to update their credentials and enforce strong passwords and limited lifetime for passwords and other tokens.
 
+**R9.11** RPP must support the Least Privilege Principle, to allow server operators to ensure that clients have only the permissions necessary.
+
+**R9.12** RPP servers SHOULD implement rate limiting and throttling to mitigate denial-of-service (DoS) attacks and abuse.
+
+**R9.13** Security-relevant events (such as failed authentication attempts, access to sensitive resources, or configuration changes) SHOULD be logged and monitored. Logs MUST NOT contain sensitive data such as passwords or full tokens.
```

Comment:
This also does not look like protocol requirement. Or is it meant that there should be security events poll queue?